testsuite/MDAnalysisTests/util.py

```diff
@@ -148,3 +149,50 @@ def in_dir(dirname):
     os.chdir(dirname)
     yield dirname
     os.chdir(old_path)
+
+
+def assert_nowarns(warning_class, *args, **kwargs):
```

Comment:
Sorry for the confusion. I was convinced we had tests for the test utils. It would be a good idea, but for an other time and PR.